server/src/test/resources/server_test_data.sql

```diff
@@ -1,104 +0,0 @@
---
```

Comment:
Yeah I think it wouldn't be a bad idea to have a ImportTestDataCommand.java or something similar.  It could be used just like [MigrateCommand.java](https://github.com/square/keywhiz/blob/master/server/src/main/java/keywhiz/commands/MigrateCommand.java)